src/components/ItemFilter/styles/FilterSettingsDialog.module.css

```diff
@@ -0,0 +1,17 @@
+.descContainer {
+    font-size: 14px;
+    line-height: 1.4;
+    margin-bottom: 16px;
```

Comment:
you can use spacers from the design system for padding and margins `var(--spacers-dp16)`

Reply:
fixed